README.md

```diff
@@ -30,89 +81,164 @@ $response = $fraudService->validateRequest($request);
+    'apiKey' => 'XXX',
+    'merchantId' => '123456',
+]);
 ```
 
-Each package already requires [`omnifraud/common`](https://github.com/lxrco/omnifraud-common),
-so you don't need to require it.
+Or, use the static `create()` method offered by `Omnifraud\Omnifraud`. The first parameter is the **alias** of the driver (detailed in the [table above](#fraud-services--drivers)), and the second is the same configuration array you would normally supply.
 
-You can also install ALL supported services:
+```php
+<?php
 
-```bash
-composer require omnifraud/omnifraud
+use Omnifraud\Omnifraud;
+
+/** @var \Omnifraud\Contracts\ServiceInterface $fraudService */
+$fraudService = Omnifraud::create('Kount', [
+    'apiKey' => 'XXX',
+    'merchantId' => '123456',
+]);
 ```
 
+As every driver is an implementation of `Omnifraud\Contracts\ServiceInterface`, you should be careful to ensure that you typehint this contract instead of any concrete implementations. Doing anything else would defeat the purpose of this library.
 
-## Fraud services
+### Front-end Implementation
 
-All drivers must implement [ServiceInterface](https://github.com/lxrco/omnifraud-common/blob/master/src/Contracts/ServiceInterface.php).
+All services expose a `trackingCode(string $pageType, string $sessionId)` method which returns a stringified snippet of JavaScript. You can call this method to insert the code necessary to instrument the front-end tracking of your fraud service. The two required parameters are a constant to specify the type of page we're inserting the snippet into, and the clients [session ID](#the-session-id).
 
-The following services are officially supported right now:
+Be sure to pass the appropriate constant as some services will differentiate between the two types of pages. It can be one of these two values:
+
+* `ServiceInterface::PAGE_ALL`
+* `ServiceInterface::PAGE_CHECKOUT`
 
-Service | Composer Package | Maintainer
---- | --- | ---
-[Kount](https://github.com/lxrco/omnifraud-kount) | omnifraud/kount | [LXRandCo](https://github.com/lxrco)
-[Signifyd](https://github.com/lxrco/omnifraud-signifyd) | omnifraud/signifyd | [LXRandCo](https://github.com/lxrco)
+Quite simply, the return string is an IIFE including the session ID. To help clarify how this works let's take a peak at the `SignifydService`. Its `trackingCode()` method will return the following JavaScript snippet:
 
+```javascript
+(function() {
+    var script = document.createElement('script');
+    script.setAttribute('src', 'https://cdn-scripts.signifyd.com/api/script-tag.js');
+    script.setAttribute('data-order-session-id', {{ $sessionId }});
+    script.setAttribute('id', 'sig-api');
+
+    document.body.appendChild(script);
+})();
+```
+
+By default, anything you pass as `$sessionId` will be quoted and escaped (via `json_encode`). If you wish to pass raw JS (i.e. a variable name, global function, etc.), take advantage of the third, optional `bool $quote = true` parameter of `trackingCode`.
```

Comment:
Nitpicking here, but we can we make it explicit that `$quote` should be `false` for the argument not to be escaped?